modules/kube-worker-sg/main.tf

```diff
@@ -0,0 +1,35 @@
+/**
+ * ## Kube Worker Security Group
+ *
+ * Define a security group for kube worker
+ *
+ */
+
```

Comment:
This is missing from the example your code is derived from, but it would make sense to add a rule for SSH to the workers, and the CIDR blocks variable for that.